packages/crypto/src/did.ts

```diff
@@ -38,6 +41,8 @@ export const formatDidKey = (jwtAlg: string, keyBytes: Uint8Array): string => {
   }
   if (jwtAlg === 'ES256') {
     keyBytes = p256.compressPubkey(keyBytes)
+  } else if (jwtAlg === 'ES256K') {
+    keyBytes = secp.compressPubkey(keyBytes)
```

Comment:
We could use the new JWT algorithm consts here.

Reply:
oh good find thx 🙏

Reply:
fixing in the other locations too 👌